community/contributing/bisecting_regressions.rst

```diff
@@ -32,20 +32,27 @@ range of commits that potentially need to be built from source and tested.
 You can find binaries of official releases, as well as alphas, betas,
 and release candidates `here <https://downloads.tuxfamily.org/godotengine/>`__.
 
-For example, if you've reported a bug against Godot 3.2, you should first try to
-reproduce the bug in Godot 3.1 (not a patch release, see below for the reason).
-If the bug doesn't occur there, try to reproduce it in Godot 3.2 *beta 1* (which
-is roughly in the middle of all test builds available). If you can't reproduce
-the bug with Godot 3.2 beta 1, then try newer betas and RC builds. If you do
-manage to reproduce the bug with Godot 3.2 beta 1, then try older alpha builds.
+If you can reproduce an issue on Godot 4.0, we recommend trying to reproduce the
+issue in the latest Godot 3.x version (if the feature exhibiting the bug is
+present in 3.x). This can be used to check whether the issue is a regression in
+4.0 or not.
+
+If the issue **is present** in 3.x, then you'll need to check whether the issue
+occurs in older 3.x versions as well.
+
+If the issue is **not present** in 3.x, then you can try older 4.0 alphas and
+betas to determine when the regression started.
```

Comment:
Maybe this should be a list? It's a bit awkward looking as individual paragraphs made out of single sentences.

For comparison:
![image](https://user-images.githubusercontent.com/11782833/209449379-316ea9a4-d040-4ce6-bfbe-5a5acb97e1e3.png)